ci/deps/actions-37-cov.yaml

```diff
@@ -51,4 +51,4 @@ dependencies:
     - brotlipy
     - coverage
     - pandas-datareader
-    - pyxlsb
+    - pyxlsb>=1.0.8
```

Comment:
users can use any version of pyxlsb but developers who run pytest locally should have pyxlsb>=1.0.8 to avoid failed tests.

Is there a better/another place where this version constraint should be specified?

1.0.8 is the first version of pyxlsb that has no internal ResourceWarnings.

Reply:
sure put it in the min versions build, and `pandas/compat/_optional.py`

Reply:
and in install.rst

Reply:
> sure put it in the min versions build

Is that azure-37-minimum_versions.yaml?

Reply:
yes